packages/server/src/common/email-templates/pioneer-email.tsx

```diff
@@ -0,0 +1,195 @@
+import { Body } from '@react-email/body'
+import { Button } from '@react-email/button'
+import { Container } from '@react-email/container'
+import { Head } from '@react-email/head'
+import { Heading } from '@react-email/heading'
+import { Html } from '@react-email/html'
+import { Img } from '@react-email/img'
+import { Link } from '@react-email/link'
+import { Preview } from '@react-email/preview'
+import { Section } from '@react-email/section'
+import { Text } from '@react-email/text'
+import * as React from 'react'
+
+import { PIONEER_URL } from '../config'
+
+interface PioneerEmailTemplateProps {
+  memberHandle: string
+  text: string
+  summary: string
+  button: {
+    href: string
+    label: string
+  }
+}
+
+export const PioneerEmailTemplate = ({
+  memberHandle = 'bob',
+  text = 'New council election has just started. Follow the link below to announce your candidacy.',
+  summary,
+  // button default value only for email development preview
+  button = {
+    href: `${PIONEER_URL}/#/election`,
+    label: 'See on Pioneer',
+  },
+}: PioneerEmailTemplateProps) => (
```

Comment:
That's a small detail but why not:
```suggestion
}: PioneerEmailTemplateProps) => render(
```
?

Reply:
good call, added exported `renderPioneerEmail` function